puppet.gemspec

```diff
@@ -29,6 +29,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency(%q<concurrent-ruby>, "~> 1.0")
   spec.add_runtime_dependency(%q<deep_merge>, "~> 1.0")
   spec.add_runtime_dependency(%q<scanf>, "~> 1.0")
+  spec.add_runtime_dependency(%q<json>, "< 2.9") # last known good version is 2.8.2
```

Comment:
2.9 introduces more breaking changes for Ruby 3.0. We package openvox7 with ruby 2.7, so probably not worth investigating.